api/scenario_api/launch/scenario_api_sample.launch.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<launch>
+  <node pkg="scenario_api" exec="scenario_api_node" name="scenario_api_sample" output="screen">
+    <remap from="/input/pointcloud" to="/sensing/lidar/no_ground/pointcloud" />
+    <remap from="/input/vectormap" to="/map/vector_map" />
+    <remap from="/input/route" to="/planning/mission_planning/route" />
+    <remap from="/input/autoware_state" to="/autoware/state" />
+    <remap from="/input/vehicle_twist" to="/vehicle/status/twist" />
+    <remap from="/input/signal_command" to="/vehicle/status/turn_signal" />
+    <remap from="/output/start_point" to="/initialpose" />
+    <remap from="/output/initial_velocity" to="/initialtwist" />
+    <remap from="/output/goal_point" to="/planning/mission_planning/goal" />
+    <remap from="/output/check_point" to="/planning/mission_planning/checkpoint" />
+    <remap from="/output/autoware_engage" to="/autoware/engage" />
+    <remap from="/output/simulator_engage" to="/vehicle/engage" />
+    <remap from="/output/npc_simulator_engage" to="/simulation/npc_simulator/engage" />
+    <remap from="/output/limit_velocity" to="/planning/scenario_planning/max_velocity" />
+    <remap from="/output/object_info" to="/simulation/npc_simulator/object_info" />
+    <remap from="/output/traffic_detection_result" to="/perception/traffic_light_recognition/traffic_light_states" />
+    <remap from="/output/lane_change_permission" to="/planning/scenario_planning/lane_driving/lane_change_approval" />
+    <remap from="/rosparam/add_simulator_noise" to="/simple_planning_simulator/add_measurement_noise" /> <!-- rosparam: add simulator noise or not -->
```

Comment:
rosparam isn't there yet anymore, is it? What should we do with this?

Reply:
it's just a comment, I would do nothing at all

Reply:
No, I mean the `from` value.

Reply:
@nnmm It's defined in `scenario_api_autoware.cpp`:

      add_simulator_noise_ = node_->declare_parameter<bool>("rosparam/add_simulator_noise", true);

Since it's just a name and doesn't rely on ROS1 rosparam functionality, I'd leave it as is